### PR TITLE
Updated Ubuntu and nginx versions.

### PR DESCRIPTION
**Pull request by bashilias**
- Moved to Ubuntu 22.04 and Nginx 1.22. 
- Added Docker install script to README.

### CODE REVIEW

**Comment by mattcen** [comment]
Any reason not to just base these Dockerfiles of https://hub.docker.com/_/nginx/ instead?
EDIT: I see this is already being suggested in #23

File: nginx-relay/Dockerfile [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
+FROM ubuntu:22.04
 
 RUN apt-get update && apt-get -y upgrade && \
     apt-get install -y wget libpcre3-dev build-essential libssl-dev zlib1g-dev && \
     rm -rf /var/lib/apt/lists/*
 
 WORKDIR /opt
 
-RUN wget https://nginx.org/download/nginx-1.18.0.tar.gz && \
+RUN wget https://nginx.org/download/nginx-1.22.0.tar.gz && \
```
Comment (martinezpl) [inline nginx-relay/Dockerfile:9]:
```suggestion
RUN wget https://nginx.org/download/nginx-1.23.1.tar.gz && \
```

File: nginx-terminate/Dockerfile [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
+FROM ubuntu:22.04
 
 RUN apt-get update && apt-get -y upgrade && \
     apt-get install -y wget libpcre3-dev build-essential libssl-dev zlib1g-dev && \
     rm -rf /var/lib/apt/lists/*
 
 WORKDIR /opt
 
-RUN wget https://nginx.org/download/nginx-1.18.0.tar.gz && \
+RUN wget https://nginx.org/download/nginx-1.22.0.tar.gz && \
```
Comment (martinezpl) [inline nginx-terminate/Dockerfile:9]:
```suggestion
RUN wget https://nginx.org/download/nginx-1.23.1.tar.gz && \
```

**Review by martinezpl — APPROVED** [review]
Bumping nginx further

**Comment by TommyTran732** [comment]
To be honest I just think it's better to use the upstream images. Bumping manually is tiring, and it's not like Signal merge pull requests for this repo often. 

Even if this does get merged, it will quickly become outdated again.

**Comment by bashilias** [comment]
Not relevant anymore.

**Pull request closed without merging**